scripts/build-wheels-macOS.sh

```diff
@@ -38,7 +37,6 @@ echo "Running $NUMPROC parallel jobs"
 for PYBIN in /Users/runner/hostedtoolcache/Python/3.*/x64/bin/python3.[5-9]
 do
     cmake .. -DPYTHON_EXECUTABLE:FILEPATH=$PYBIN \
-             -DCMAKE_MACOSX_RPATH=ON \
```

Comment:
Default value of MACOSX_RPATH is TRUE (policy [CMP0042](https://cmake.org/cmake/help/latest/policy/CMP0042.html#policy:CMP0042))